tests/unit/test_indicator.py

```diff
@@ -0,0 +1,415 @@
+'''
+Workflow Serialization Unit Tests
+
+To run unittests:
+
+# Using standard library unittest
+
+python -m unittest -v
+python -m unittest tests/unit/test_indicator.py -v
+
+or
+
+python -m unittest discover <test_directory>
+python -m unittest discover -s <directory> -p 'test_*.py'
+
+# Using pytest
+# "conda install pytest" or "pip install pytest"
+pytest -v tests
+pytest -v tests/unit/test_indicator.py
+
+'''
+import os
+import warnings
+from io import StringIO
+import shutil, tempfile
+import pandas as pd
+import unittest
+import pathlib
+import cudf
+from difflib import context_diff
+from gquant.cuindicator import (rate_of_change, bollinger_bands, trix, macd,
+                                average_true_range, ppsr,
+                                stochastic_oscillator_k,
+                                stochastic_oscillator_d,
+                                average_directional_movement_index,
+                                vortex_indicator,
+                                kst_oscillator, relative_strength_index,
+                                mass_index, true_strength_index,
+                                chaikin_oscillator,
+                                money_flow_index, on_balance_volume,
+                                force_index, ease_of_movement,
+                                ultimate_oscillator, donchian_channel,
+                                keltner_channel, coppock_curve,
+                                accumulation_distribution,
+                                commodity_channel_index, moving_average,
+                                exponential_moving_average, momentum)
+
+from . import technical_indicators as p
+import numpy as np
+# --------------------------------------------------------- Keep tests in order
+def make_orderer():
```

Comment:
The `make_ordered` should be moved into a `testutils.py` module and re-used throughout to adhere to DRY principle.

Reply:
Will do